templates/bundles/DemosPlanCoreBundle/DemosPlanProcedure/administration_statement_segments_list.html.twig

```diff
@@ -13,6 +13,7 @@
 {% block component_part %}
     <statement-segments-list
         :current-user="JSON.parse('{{ user|json_encode|e('js', 'utf-8') }}')"
+        :is-source-and-coupled-procedure="{{ templateVars.isSourceAndCoupledProcedure == 1 ? 'true' : 'false' }}"
```

Comment:
Doesn't `{{ templateVars.isSourceAndCoupledProcedure }}` do the trick as well?

Reply:
thanks done https://github.com/demos-europe/demosplan-core/pull/2549/commits/c4fc0363d3264411df7471d54a2d797781273aa5

Reply:
Did you try it? Only `{{ templateVars.isSourceAndCoupledProcedure }}` did not work? Perhaps then as `is-source-and-coupled-procedure="{{ templateVars.isSourceAndCoupledProcedure }}"` (without `:`) But I am fine with it as is

Reply:
ah yes, I see. sorry didnt recognized it here

Reply:
okay, I tried it like your suggestion but it didnt work tthis way